lib/utils.js

```diff
@@ -0,0 +1,11 @@
+var path = require('path');
+
+/* Normalize URL or path to an absolute path, if it corresponds to one. */
+function filePath(urlOrPath) {
+  // give naked paths the benefit of the doubt that they are file paths
+  if (path.isAbsolute(urlOrPath))
```

Comment:
Yeah the `path.isAbsolute('http://site.com') === false` is assumed here - the purpose of the function is to return a path, if the input looks like a path, or is a file URL. `'http://site.com` is neither, so it would fail both `if`s and return something falsey (`undefined`).